src/Symfony/Component/VarDumper/Dumper/HtmlDumper.php

```diff
@@ -117,7 +137,8 @@ protected function getDumpHeader()
             return $this->dumpHeader;
         }
 
-        $line = <<<'EOHTML'
+        $line = json_encode($this->displayOptions, JSON_FORCE_OBJECT);
+        $line = <<<EOHTML
```

Comment:
We have to stick with nowdoc here or alternatively have to escape `$` characters below.